cmd/system-probe/modules/network_tracer.go

```diff
@@ -119,6 +119,19 @@ func (nt *networkTracer) Register(httpMux *module.Router) error {
 		utils.WriteAsJSON(w, debugging.HTTP(cs.HTTP, cs.DNS))
 	})
 
+	httpMux.HandleFunc("/debug/ebpf_maps", func(w http.ResponseWriter, req *http.Request) {
+		ebpfMaps, err := nt.tracer.DebugEBpfMaps()
+		if err != nil {
+			log.Errorf("unable to retrieve eBPF maps: %s", err)
+			w.WriteHeader(500)
+			return
+		}
+
+		contentType := req.Header.Get("Accept")
+		marshaler := encoding.GetMarshaler(contentType)
+		writeConnections(w, marshaler, ebpfMaps)
```

Comment:
I don't think this will work because `writeConnections` is expecting a `*network.Connections`

Reply:
I should flag as wip as it wasn't compiled